feat: implement user research banner (#756)
commercetools/commercetools-docs-kit: .changeset/six-trains-switch.md

---

> '@commercetools-docs/gatsby-theme-docs': patch

Reviewer: A new but backwards compatible feature is a minor, not a patch. 
https://semver.org/

Author: Thanks for the link 👍

> '@commercetools-docs/gatsby-theme-docs': minor
> '@commercetools-docs/ui-kit': patch

Reviewer 2: ```suggestion
'@commercetools-docs/gatsby-theme-docs': minor
```

---

> A user research banner was implemented for all sites

Reviewer 2: ```suggestion
Render a User Research banner in all content pages.
```
